onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -3076,5 +3076,267 @@ TEST(BitShiftOpTest, BroadcastXRight_Uint8) {
   test.Run();
 }
 
+TEST(MathOpTest, BitwiseAnd) {
```

Comment:
Hmm, it saddens me to see c++ code written to test these than *only* exist within ORT, rather than data-driven .onnx files which would benefit *everybody* implementing the standard (e.g. https://github.com/onnx/onnx/tree/main/onnx/backend/test/data/node/test_bitwise_not_2d) or JSON files which aren't C++ specific (like our WindowsAI conformance test suite, which I pointed the task to Numfor about). Of course, I'm still will gladly accept additional test collateral, but it's current form doesn't have the biggest impact/benefit - something to think about...